cli/command/container/cp.go

```diff
@@ -272,13 +362,28 @@ func copyToContainer(ctx context.Context, dockerCli command.Cli, copyConfig cpCo
 
 		resolvedDstPath = dstDir
 		content = preparedArchive
+		copySize, content = getSize(content)
```

Comment:
Shouldn't we already have the size of this?

Reply:
No, because the file props are obtained during the methods `copyFromContainer` and `copyToContainer`. But I'll remove this and only show the amount transferred.

Reply:
We should be able to get size, for local files as well as remote (though I'm not sure if the remote stat returns size right now).

Reply:
I have changed this. Now, only the transferred amount is displayed.